helm/aladdin-demo/templates/shared/nfs-mount.pv.yaml

```diff
@@ -0,0 +1,16 @@
+{{ if .Values.deploy.withMount }}
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ .Chart.Name }}-nfs-volume
+spec:
+  capacity:
+    storage: 1Gi
+  accessModes:
+  - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: standard
+  nfs:
+    server: 192.168.99.1
```

Comment:
Anyway we can auto-populate this with `$(minikube ip)`?

Reply:
I'm not 100% on this, but this isn't the minikube ip, rather the ip of your host machine from inside the container, AFAIK that shouldn't change

Reply:
It looks like `minikube-nfs` is calling that directly when it makes the mount request though.